.github/workflows/cml.yaml

```diff
@@ -4,20 +4,29 @@ jobs:
   run:
     runs-on: [ubuntu-latest]
     container: docker://dvcorg/cml:0-dvc2-base1
+    env:
+      GIT_USER: dvc
+      GIT_EMAIL: dvc@iterative.ai
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
+          ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 0
+      - name: Fix git safe.directory in container
+        run: |
+          git config --global --add safe.directory ${GITHUB_WORKSPACE}
+          git config --global user.name ${GIT_USER}
+          git config --global user.email ${GIT_EMAIL}
       - name: Generate metrics report
         env:
           REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           pip install -r requirements.txt
-          
-          dvc get https://github.com/iterative/dataset-registry/ dvc-course/hymenoptera_data
 
+          dvc get https://github.com/iterative/dataset-registry/ dvc-course/hymenoptera_data
```

Comment:
Not sure what this line is supposed to do as the downloaded folder is not used by the pipeline in `dvc.yaml`?